src/utils.ts

```diff
@@ -96,7 +96,22 @@ export const normalizeDirectionParams = ({
         return {
             limit: Math.max(1, last ?? 20),
             cursor: before ? decodeCursor(before) : null,
-            sort: mapValues(sort, (value) => value * -1),
+            sort: mapValues(
```

Comment:
Could you ad a comment describing why we reverse these values? Without any explanation, it looks like a bug/typo

Reply:
I've actually no idea what the logic is tbqh; my goal is merely to make the existing code work with the new mongodb version :)

Reply:
Ah yes sorry I didn't realize that logic was already there. All good!